lib/poll.js

```diff
@@ -3,8 +3,8 @@ const crypto = require('crypto');
 function Poll() {
   this.id = crypto.randomBytes(20).toString('hex');
   this.voterId = crypto.randomBytes(20).toString('hex');
-  this.adminUrl = `http://localhost:3000/${this.id}`;
-  this.voterUrl = `http://localhost:3000/${this.voterId}`;
+  this.adminUrl = `http://localhost:3000/admin/${this.id}`;
+  this.voterUrl = `http://localhost:3000/vote/${this.voterId}`;
```

Comment:
We probably don't want to hard-code in `http://localhost:3000`. These will break in production. Also, they're not super dry. If for some reason down the road `this.id` or `this.voterId` changed (Imagine the user sent out the wrong one and wanted to reset their `id`—this isn't super uncommon), then you'd have to change this in two places.

Alternatively, you could use methods here:

```js
Poll.prototype.adminUrl = function () {
  return '/polls` + this.id;
}
```